grakn-engine/src/main/java/ai/grakn/engine/bootup/config/StorageConfig.java

```diff
@@ -91,9 +95,11 @@ private StorageConfig updateDirs(GraknConfig config) {
         String dbDir = config.getProperty(GraknConfigKey.DATA_DIR);
 
         ImmutableMap<String, Object> dirParams = ImmutableMap.of(
-                DATA_FILE_DIR_CONFIG_KEY, Collections.singletonList(dbDir + DATA_SUBDIR),
-                SAVED_CACHES_DIR_CONFIG_KEY, dbDir + SAVED_CACHES_SUBDIR,
-                COMMITLOG_DIR_CONFIG_KEY, dbDir + COMMITLOG_SUBDIR
+                DATA_FILE_DIR_CONFIG_KEY, Collections.singletonList(dbDir + DATA_FILE_DIR_CONFIG_VALUE),
```

Comment:
The `data-dir` parameter in `grakn.properties` is now correctly appliad to Cassandra's `cdc_raw_directory` and `hints_directory`